README.md

```diff
@@ -55,9 +55,25 @@ The build process is managed with [Make](https://www.gnu.org/software/make/).
 * `delete`: Remove the application from the device
 * `build`: Build obj and bin api artefacts without loading
 * `format`: Format source code.
+* `analyze`: Run clang static analyzer (requires clang-tools)
+* `size`: Prints the app size.
 
 See `Makefile` for list of included functions.
 
+`make load` results in an exception in the following cases (there might be others):
+* the Ledger device is not connected
+* the device is locked
+* the device is in some intermediary state (e.g. processing an APDU message)
+* the app is too big to fit on the device (try `make size`)
+
+### Before merging a pull request
+
+_Before merging a PR, one should make sure that:_
+* `make format` does not change anything (except possibly some glyph* files)
+* `make clean load` runs without errors and warnings (except those reported for nanos-secure-sdk repo) for production build
+* `make clean load` runs without errors and warnings (except those reported for nanos-secure-sdk repo) for development build (see Debug version above)
+* `make analyze` does not report errors or warnings
```

Comment:
This will be tricky, I can't yet figure out how to blacklist the SDK folder or whitelist the project source. And the SDK has a problem with the snprintf (what else).

Reply:
I keep the SDK separated, but if you figure out how to make the command more specific to the project source, let's use it.

Reply:
Ah yeah, it's my bad, I experimented with various SDKs (for the new, docker based build recommended by the ledger dev website), and one was left in the ledger directory.